fastserde/avro-fastserde-tests-common/src/test/avro/logicalTypesTest1.avsc

```diff
@@ -0,0 +1,203 @@
+{
+  "type": "record",
+  "name": "FastSerdeLogicalTypesTest1",
+  "namespace": "com.linkedin.avro.fastserde.generated.avro",
+  "doc": "Used in tests to confirm fast-serde supports logical-types",
+  "fields": [
+    {
+      "name": "enumField",
+      "type": {
+        "name": "JustSimpleEnum",
+        "type": "enum",
+        "symbols": [
+          "E1",
+          "E2",
+          "E3",
+          "E4",
+          "E5"
+        ]
+      }
+    },
+    {
+      "name": "arrayOfEnums",
+      "type": [
+        "null",
+        {
+          "type": "array",
+          "items": "JustSimpleEnum"
+        }
+      ],
+      "default": null
+    },
+    {
+      "name": "mapOfEnums",
+      "type": {
+        "type": "map",
+        "values": "JustSimpleEnum"
+      }
+    },
+    {
+      "name": "unionOfArrayAndMap",
```

Comment:
For the sake of completeness - how about the map with values of unions (with logical types)?

Reply:
Added in commit `[review-fixes] Part 4`